src/details/ArborX_DetailsDistributedTreeNearestHelpers.hpp

```diff
@@ -110,8 +110,7 @@ struct approx_expand_by_radius<PointTag, Point>
   {
     constexpr int DIM = GeometryTraits::dimension_v<Box>;
     using Coordinate = GeometryTraits::coordinate_type_t<Point>;
-    return ExperimentalHyperGeometry::Sphere<DIM, Coordinate>{
-        Kokkos::bit_cast<::ArborX::Point<DIM, Coordinate>>(point), r};
+    return Sphere{Kokkos::bit_cast<::ArborX::Point<DIM, Coordinate>>(point), r};
```

Comment:
Is the `bit_cast` still necessary here?

Reply:
Yes. This geometries may come from users. We do an implicit undocumented cast here as we don't have a well defined concept of a point. It's ok for now but will need to change in the future.